Gemfile.lock

```diff
@@ -240,6 +244,7 @@ GEM
 
 PLATFORMS
   ruby
+  x64-mingw32
```

Comment:
I'm not sure what the changes in this file are about, or whether GitHub Pages will like them. Did you do this work on a Windows machine?

Reply:
Oh yes, that shouldn't have been there, I'll remove it before merging.